packages/integrations/image/src/build/ssg.ts

```diff
@@ -83,10 +83,10 @@ export async function ssgBuild({ loader, staticImages, config, outDir, logLevel
 			let outputFile: string;
 
 			if (isRemoteImage(src)) {
-				const outputFileURL = new URL(path.join('./', path.basename(filename)), outDir);
+				const outputFileURL = new URL(path.join('./assets', path.basename(filename)), outDir);
```

Comment:
Can I have a custom `outDir` like `dist/assets/img/`? Seems like I'd want to control that so I don't always get `dist/assets/img/assets/...`?

Reply:
Yep I definitely want to add that config, I had considered doing it in this PR but wanted to get the bug fix out the door without blocking on a new feature.

Happy to revisit that and add a config option here if you think it's worth doing both here though 👍

Reply:
Fine with a quick bugfix to get out and revisiting this in a follow-up!